index.ts

```diff
@@ -29,11 +29,33 @@ export default class AuditLogPlugin extends AdminForthPlugin {
 
   static defaultError = 'Sorry, you do not have access to this resource.'
 
+  private persistLogRecord = async (
```

Comment:
@tesaide could you please explain how this change is related to link creation on show and list view?